pkg/api/v1/conversion.go

```diff
@@ -740,6 +734,7 @@ func Convert_v1_PodSecurityContext_To_api_PodSecurityContext(in *PodSecurityCont
 	return nil
 }
 
+// +k8s:conversion-fn=copy-only
```

Comment:
Aah - it seems to be related to another commit, where you moved non-trivial logic to defaulting, right?